include/ajax.search.php

```diff
@@ -391,6 +393,16 @@ function collectQueueCounts($ids=null) {
         if (!$thisstaff)
             Http::response(403, 'Agent login is required');
 
+        $queues = CustomQueue::objects()
+            ->filter(Q::any(array(
+                'flags__hasbit' => CustomQueue::FLAG_PUBLIC,
+                Q::any(array(
+                    'flags__hasbit' => CustomQueue::FLAG_VISIBLE_TEAM,
+                    'team__members__staff_id' => $thisstaff->getId(),
+                )),
+                'staff_id' => $thisstaff->getId(),
+            )));
+
```

Comment:
Oops. Looks like `$queues` is not used here and might need to be moved to `SavedQueue::ticketsCount`